spec/controllers/availabilities/places_controller_spec.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Availabilities::PlacesController, type: :controller do
+  describe 'GET #index' do
+    context 'unauthenticated' do
+      it do
```

Comment:
Why use `context` when you leave the `it`s blank? Why not just put it in the `it`s?